compiler/env/jittypes.h

```diff
@@ -124,8 +124,7 @@ typedef struct TR_ByteCodeInfo
 
 typedef struct TR_InlinedCallSite
    {
-   TR_OpaqueMethodBlock   * _methodInfo;
-   #define _vmMethodInfo _methodInfo
+   TR_OpaqueMethodBlock   * _method;
```

Comment:
This is problematic for DDR in OpenJ9: it will break access to core files created before this change.

Reply:
Is the problem just with removing `#define`? If it is, I can leave it in. Or is it that we are now not storing `TR_AOTMethodInfo *` at `_methodInfo`?

Reply:
I think it's  specifically renaming `_methodInfo`; the define just gets expanded out to `_methodInfo` anyway in the preprocessor.

Reply:
Also I believe DDR only uses this for reading from the J9JITExceptionTable struct; it does not read it out from the comp object.

Reply:
Whether the macro `_vmMethodInfo` stays or goes only affects C/C++ code that uses it. If there are no (more) uses, it can safely be removed.

Reply:
ok, then I will just keep the name as `_methodInfo`.